tests/uid2_token_generator.py

```diff
@@ -0,0 +1,105 @@
+import base64
```

Comment:
this file will eventually be moved under uid2_client when @jon8787 gets to implement uid2 sharing in this py client.

Reply:
think @zaiweidu and @cody-constine-ttd are working on Sharing in python

Reply:
ok